package.json

```diff
@@ -66,6 +66,8 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
It's safe to remove `dependencies` if needed. Some of my libraries do that and it works just fine.